include/amici/rdata.h

```diff
@@ -101,12 +101,16 @@ class ReturnData {
      * nplist x ny, row-major) */
     std::vector<realtype> ssigmay;
     
-    /** observable (dimension: nt x ny, row-major) */
+    /** observable (dimension: nt*ny) */
```

Comment:
still row-major?

Reply:
its 1D ...

Reply:
If it is nt*ny it still need to be clear if e.g. the second element corresponds to (t0,y1) or (t1,y0), not?